src/templates/robots.ts

```diff
@@ -0,0 +1,30 @@
+import type {
+  TemplateProps,
+  TemplateRenderProps,
+  GetPath,
+  TemplateConfig,
+} from "@yext/pages";
+
+export const config: TemplateConfig = {
+  // The name of the feature. If not set the name of this file will be used (without extension).
+  // Use this when you need to override the feature name.
+  name: "robots",
+};
+
+/**
+ * Defines the path that the generated file will live at for production.
+ *
+ * NOTE: This currently has no impact on the local dev path. Local dev urls currently
+ * take on the form: featureName/entityId
+ */
+export const getPath: GetPath<TemplateProps> = () => {
+  return `robots.txt`;
+};
+
+export const render = (data: TemplateRenderProps) => {
+  console.log(data.document);
+  return `User-agent: *
+Disallow:
```

Comment:
this is RP so we want to disallow / right? And they'll handle stuff on their end?

Reply:
Yes I think so. [This](https://docs.google.com/document/d/1HdJAQzYBqx5oms5vv6xIpGxnDPhUTKLEktrinp_wlEE/edit#heading=h.qpumyfq7pma0) is their RP instructions doc, and it says `As of right now, there is nothing that we need to block on the robots.txt file for the pros.turbotax.intuit.com subdomain. 
`

Reply:
rn it looks like it's not disallowing anything. I'm also a little confused why they want us to place the meta tags individually on every page instead of handling it all through a single robots.txt that they control. This is maybe something we want to sync with their developers on

Reply:
That's a valid point, I was also confused why they wanted robots.txt if they already have robots meta tags on every page. I'll let PC know.